index.html

```diff
@@ -4817,6 +4789,73 @@ <h2>Content Integrity Protection</h2>
 hash for the context as registered in the DID specification registries
 (see the registration process for more detail) [[?DID-SPEC-REGISTRIES]].
       </p>
+    <section>
+      <h2>Persistence</h2>
+      <p>
+    </section>
```

Comment:
Joe, for some reason when I preview this PR, this section appears as a subsection of the previous section instead of a sibling to it. But that might be a preview bug because the section tag appears correct.

In any case, you might also consider making it section 9.9, i.e., right after 9.8 "The Role of Human-Friendly Identifiers", as it seems to fit right before "Immutability of DID Documents".

Reply:
Good catch. There was a </section> tag in the wrong place.